spec/features/aquariums/index_spec.rb

```diff
@@ -12,8 +12,10 @@
   scenario 'visitor sees name of each aquarium record in system' do
     visit '/aquariums'
 
+  expect(current_path).to eq("/aquariums")
   expect(page).to have_content(@shedd.name)
   expect(page).to have_content(@national.name)
   expect(page).to have_content(@denver.name)
+  expect(page).to_not have_content(@shedd.clean)
```

Comment:
Do we have to test all variables that could be contained? Or just one?

Reply:
Theoretically just one.